components/brave_ads/browser/ads_notification_handler.h

```diff
@@ -107,7 +107,6 @@ class AdsNotificationHandler : public NotificationHandler {
 
   std::map<std::string, base::OnceClosure> pending_close_callbacks_;
   content::BrowserContext* browser_context_;
-  brave_ads::AdsServiceImpl* ads_service_;
```

Comment:
We should not use a global, please revert to use the member variable and the setter